src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SchedulingAndListingFields.java

```diff
@@ -1,24 +1,22 @@
-package uk.gov.hmcts.reform.sscs.ccd.domain;
-
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-import com.fasterxml.jackson.annotation.JsonInclude;
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.Data;
-import lombok.NoArgsConstructor;
-
-@JsonIgnoreProperties(ignoreUnknown = true)
-@Data
-@Builder(toBuilder = true)
-@AllArgsConstructor
-@NoArgsConstructor
-@JsonInclude(JsonInclude.Include.NON_NULL)
-public class SchedulingAndListingFields {
-    private HearingRoute hearingRoute;
-
-    private HearingState hearingState;
-
-    private Long activeHearingId;
-
-    private Long activeHearingVersionNumber;
-}
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class SchedulingAndListingFields {
+    private HearingRoute hearingRoute;
```

Comment:
Has the line endings been changed on this file always use LF as its saying the whole file has changed when it hasn't also id keep the space between the variable looks a bit nicer and cleaner

Reply:
Hmm good shout - I have just checked and on my local they are LF